README.md

```diff
@@ -152,6 +152,15 @@ translations_list = [
 content_prefix = "ecosystem/finance"
 ```
 
+## Use custom RST directives
+
+The `qiskit_sphinx_theme` extension defines the below custom directives for you to use in RST, if you'd like. See `example_docs/docs/sphinx_guide/custom_directives.rst` for examples of how to use them.
+
+* `customcardelement`
+* `customcalloutitem`
```

Comment:
Is this change going to break something?

Reply:
No because the projects will still have their custom directives activated. When we switch them to these directives, we will delete their custom directives and update their RST files.